### DISCUSSION TITLE
Karte teilen - Hinweis Kartenaufruf

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#122] [Q&A]
Hallo Jürgen!

Einige User haben bei uns eine Karte mittels Werkzeug "Karte teilen" für 365 Tage geteilt. Seit kurzem kommt eine Hinweismeldung, dass der Kartenaufruf noch (wie z.B. im angehängten Beispiel) 113 Tage, 22 Stunden und 10 Minuten gültig ist. 
![hinweis_karte_teilen](https://github.com/gis-eni/webgis-forum/assets/122986367/89845b21-717a-41df-856b-4d4fd8683ffb)

Ab welchem Zeitpunkt erscheint diese Meldung? Und gibt es eine Möglichkeit die Zeit, ab der der Hinweis erscheint zu reduzieren bzw. über die api.config einzustellen?

Danke!
Lg, Andi

**Comment by gis-eni** [comment]
:) Wie ausschaut, kommt die Meldung, wenn die Hälfte der Zeit abgelaufen ist.
Sollten wir einmal ändern... Es war wohl nicht vorgesehen, dass Links so lange aufbehalten werden.

**Comment by gis-eni** [comment]
Ab Build 5.23.5001 kommt diese Warnung erst, wenn die Halbzeit erreicht ist **UND weniger als eine Woche übrig bleibt**.
